### PR TITLE
Move AlphaRatio provider to sickrage.providers module

### PR DESCRIPTION
**Pull request by MGaetan89**
Don't merge until https://github.com/SickRage/SickRage/pull/471 and https://github.com/SickRage/SickRage/pull/472 are merged. They are not technically linked, but some files are common between all those PRs.

@miigotu Do you prefer to have all the providers files in `sickrage.providers` or `sickrage.providers.(torrent|nzb)` (depending on the type of provider, of course)?
Do you see something else that should be changed? It's better to agree on everything now, before I move to the other providers.

### CODE REVIEW

**Comment by SickRage2** [comment]
Thanks for the pull request! Before a real human comes by, please make sure your PR has all of the below criteria checked
- [x] Give a description on what the PR is for.
- [x] Make sure your PR is based on the DEVELOP branch
- [x] Don't send big changes all at once. Split up big PRs into multiple smaller PRs that are easier to manage and review

Please make sure you also read [contribution guide](https://github.com/SickRage/SickRage/blob/master/contributing.md) and followed all of the steps.

Thanks!

File: sickbeard/providers/__init__.py [hunk 1]
```diff
@@ -25,18 +25,21 @@
...
     'torrentday', 'hdbits', 'hounddawgs', 'iptorrents', 'omgwtfnzbs',
     'speedcd', 'nyaatorrents', 'animenzb', 'torrentbytes', 'freshontv', 'titansoftv',
-    'morethantv', 'bitsoup', 't411', 'tokyotoshokan', 'alpharatio',
+    'morethantv', 'bitsoup', 't411', 'tokyotoshokan',
     'shazbat', 'rarbg', 'tntvillage', 'binsearch', 'bluetigers', 'cpasbien',
     'fnt', 'xthor', 'scenetime', 'btdigg', 'transmitthenet', 'tvchaosuk',
     'torrentproject', 'extratorrent', 'bitcannon', 'torrentz', 'pretome', 'gftracker',
     'hdspace', 'newpct', 'elitetorrent', 'bitsnoop', 'danishbits', 'hd4free'
+] + [
+    'AlphaRatio'
```
Comment (MGaetan89) [inline sickbeard/providers/__init__.py:42]: I'm doing this so it's easy to see which providers are in `sickbeard.providers` (first list) or in `sickrage.providers` (second list).

**Comment by labrys** [comment]
@MGaetan89 iirc miigotu wanted to separate torrent and nzb providers into their own subfolders

**Comment by miigotu** [comment]
I also want to make some changes to the providers themselves, and if you move them I won't have an easy way to keep track of which ones I have done.

I was waiting for the last changes to stabilize before moving forward with this, we still have some bugs introduced in the previous few releases

**Comment by MGaetan89** [comment]
I thought you were done with the changes, sorry.
Maybe we can keep this PR open until your are done, and then I'll update it.

**Comment by miigotu** [comment]
I want to change the `item = (some, implicit, tuple, order)` for results into a dict with explicitly expected keys, and make every provider follow this format, or use the SearchResult class and create it in each provider for each result, so we always have a simple way of accessing it rather than a blind indexing of it later.

**Comment by miigotu** [comment]
We will handle this after the providers are finished =P

**Pull request closed without merging**